news/2025/2025-01-23-osumania-7k-world-cup-2025-round-of-32-recap.md

```diff
@@ -0,0 +1,186 @@
+
+Saying that the tiebreaker was hard, would be an understamenet. Despite fighting tooth and nail, France managed to stay on top with their lead in ratio. While France may be going into the next round facing ::{ flag=CN }:: China, Macau's surely left a mark with their impressive performance this match, having to face ::{ flag=MX }:: Mexico in the round to come.
+
+### ::{ flag=VE }:: Venezuela vs. ::{ flag=CA }:: Canada
+
+<style>
+    .news-chat-quote__avatar {
+        float: left;
+        width: 40px;
+        height: 40px;
+        border-radius: 50%;
+        margin-left: -50px;
+    }
+
+    .news-chat-quote__text-container {
+        margin-left: 50px;
+    }
+
+    .news-chat-quote__username {
+        font-weight: 600;
+        margin-bottom: 2px;
+    }
+
+    .news-chat-quote__colour-nat {
+        color: #fa3703;
+    }
+
+    .news-chat-quote__colour-no-group {
+        color: #FFFFFF;
+    }
+</style>
+
+**[You can view the match information here!](https://osu.ppy.sh/community/matches/116874748)**\
+**[You can view the match VOD here!](https://www.twitch.tv/videos/2357969267)**
+
+As the week came to a close, there was but one more surprise awaiting us in the match against ::{ flag=VE }:: Venezuela (#12) and ::{ flag=CA }:: Canada (#21)! Everyone's expectations were sky-high, as this was the first time Venezuela had the chance to go against Canada in a World Cup match, who reached Semifinals just last year. Nobody expected the roller coaster of emotions to ensue, ending in a nerve-wracking tiebreaker to decide the victor.
+
+The first map chosen was [Tedjimo yomigY - Usuva Kagerow](https://osu.ppy.sh/beatmapsets/2307475#mania/4935023) (HB1). A grueling battle of precision, only won by a fairly narrow margin of 5,360 points in favour of Venezuela managing to maintain their cool till the very end. A testament to Venezuela's resilience and perseverance right off the bat! The next three maps also ended in victory for Venezuela, who extended their lead by such large, seemingly uncontested margins. At this point, many viewers thought the match would end in a complete sweep in favour of Venezuela.
+
+But lo and behold, everything changed when Canada took a gamble and chose [Neko Hacker - Shadows feat. Such](https://osu.ppy.sh/beatmapsets/2307450#mania/4934981) (LN1), a map that featured LN category not seen in previous tournaments. This proved to be very effective however, as the map's style played very well with Canada's strengths, managing to secure their first win of the match! This substantially boosted the team's morale as they managed to win point after point thereafter. Despite the multiple close calls experienced by Canada, they managed to tie things up with Venezuela, forcing a tiebreaker to be played.
```

Comment:
```suggestion
But lo and behold, everything changed when Canada took a gamble and chose [Neko Hacker - Shadows feat. Such](https://osu.ppy.sh/beatmapsets/2307450#mania/4934981) (LN1), a map that featured the new LN inverses not seen in previous tournaments. This proved to be very effective however, as the map's style played very well with Canada's strengths, managing to secure their first win of the match! This substantially boosted the team's morale as they managed to win point after point thereafter. Despite the multiple close calls experienced by Canada, they managed to tie things up with Venezuela, forcing a tiebreaker to be played.
```

Reply:
Again as a prior correction, using the term like this does sound weird. Corrected a bit.

```suggestion
But lo and behold, everything changed when Canada took a gamble and chose [Neko Hacker - Shadows feat. Such](https://osu.ppy.sh/beatmapsets/2307450#mania/4934981) (LN1), a map that featured the new LN inverse category not seen in previous tournaments. This proved to be very effective however, as the map's style played very well with Canada's strengths, managing to secure their first win of the match! This substantially boosted the team's morale as they managed to win point after point thereafter. Despite the multiple close calls experienced by Canada, they managed to tie things up with Venezuela, forcing a tiebreaker to be played.
```